js/data/buffer/collision_box_vertex_buffer.js

```diff
@@ -1,37 +1,48 @@
 'use strict';
 
 var util = require('../../util/util');
-var Buffer = require('./buffer');
-
-module.exports = CollisionBoxVertexBuffer;
-
-function CollisionBoxVertexBuffer(buffer) {
-    Buffer.call(this, buffer);
+var Buffer2 = require('../buffer2');
+
+function CollisionBoxVertexBuffer(options) {
+    Buffer2.call(this, options || {
+        type: Buffer2.BufferType.VERTEX,
+        attributes: {
+            coordinates: {
+                type: Buffer2.AttributeType.SHORT,
+                components: 2
+            },
+            extrusions: {
+                type: Buffer2.AttributeType.SHORT,
+                components: 2
+            },
+            maxZoom: {
+                type: Buffer2.AttributeType.UNSIGNED_BYTE
+            },
+            placementZoom: {
+                type: Buffer2.AttributeType.UNSIGNED_BYTE
+            }
+        }
```

Comment:
Ack. Sorry, I know this is different from what I said originally, but these names should match the actual shader attribute names where possible. So this should be `pos` and `data` as defined [here](https://github.com/mapbox/mapbox-gl-js/blob/vertex-buffers/js/render/draw_collision_debug.js#L21) and [here](https://github.com/mapbox/mapbox-gl-js/blob/vertex-buffers/shaders/collisionbox.vertex.glsl#L1)